### PR TITLE
squash form_processor migrations

### PR DESCRIPTION
**Pull request by snopoke**
Part 1 of the 2 part process: https://docs.djangoproject.com/en/1.8/topics/migrations/#squashing-migrations

:blowfish: 

@NoahCarnahan 
cc @benrudolph

### CODE REVIEW

File: corehq/form_processor/migrations/0002_drop_like_indexes.py [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+            sql='DROP INDEX IF EXISTS form_processor_commcarecasesql_case_id_3dee99b8828d1543_like',
+            reverse_sql='SELECT 1',
+            state_operations=None,
+        ),
+        migrations.RunSQL(
+            sql='DROP INDEX IF EXISTS form_processor_casetransaction_case_id_6c13626ba79fe02e_like',
+            reverse_sql='SELECT 1',
+            state_operations=None,
+        ),
+    ]
```
Comment (snopoke) [inline corehq/form_processor/migrations/0002_drop_like_indexes.py]: These had to be separate

**Comment by snopoke** [comment]
Just remembered I've left something out

**Comment by snopoke** [comment]
should be gtg once tests pass

**Comment by snopoke** [comment]
rebased off master, should fix tests

**Comment by snopoke** [comment]
:white_check_mark:

**Comment by benrudolph** [comment]
good cleanup

**Pull request merged**